src/data/vortex.graphql

```diff
@@ -137,6 +166,7 @@ type PartnerStats {
   Number of unique visitors
   """
   uniqueVisitors(period: QueryPeriodEnum!): Int
+    @deprecated(reason: "Use audience() { uniqueVisitors } instead")
```

Comment:
just to double check - does the use of `@deprecated` stop schema changes from breaking the services that use it, or will we need to update Reaction/Force/Volt schemas with this change?

Reply:
I think this is to avoid breaking changes, but we can double check locally.

Reply:
My guess is that this directive offers a warning to clients about this field, and leaves it up to users of the schema to heed or ignore it. In my experience with Java, any instantiation/call/access to a class/method/field that is annotated with `@deprecated` throws a warning in the development IDE without preventing those actions.

I've reached out to the #graphql channel because I'm not finding search results that explain what happens as a result of applying this directive to a GraphQL schema.

Reply:
I think @iskounen is right, all `@deprecated` does is warn the developer that the field is deprecated. It does not restrict usage, but serves as a warning for clients using that field. For example, there's a small tooltip that pops up when querying vortex from insomnia.
<img width="712" alt="Screen Shot 2019-06-03 at 8 01 27 AM" src="https://user-images.githubusercontent.com/1389948/58890757-15ca9c80-86a0-11e9-91f2-090c09812ffe.png">
This may be confusing because I had made another change to vortex that removed the resolver for the top level `uniqueVisitors` field, so now that field returns `null`. If I did not remove this resolver, this field would continue to work as normal, but that deprecation message would continue to pop up.

Below, I stubbed a resolver function for `uniqueVisitors` to always return `12345`.
<img width="251" alt="Screen Shot 2019-06-04 at 8 15 10 AM" src="https://user-images.githubusercontent.com/1389948/58891172-e4060580-86a0-11e9-9acc-e2b61fc5d90c.png">
And the `uniqueVisitors` field would return that value along with the deprecation message.
<img width="710" alt="Screen Shot 2019-06-04 at 8 14 16 AM" src="https://user-images.githubusercontent.com/1389948/58891093-c3d64680-86a0-11e9-8bb0-0bc4ddb8238e.png">

Reply:
Thanks for clarifying that, @ansor4.

@mzikherman made a good point about whether we should be deprecating a new field, which made me think that it may be better to rename this field in Vortex and propagate the changes through Volt at this stage of development. If we deprecate it now we'll need to eventually _actually_ deprecate it, so we may as well do that work now. What do you all think?

cc: @sepans @mdole

Reply:
My only concern would be if volt is still expecting a `uniqueVisitors` field. I _think_ volt is equipped to handle `null` as a value for `uniqueVisitors`, I'm not sure if it's equipped to not expect the field at all. @sepans I believe you're more familiar with the error handling on volt's side, would you know if this field is ok to deprecate now?

Reply:
I believe so. Going to merge this and will PR a fix if volt breaks (I think that is faster) 😎